docs/.vitepress/config.ts

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from 'vitepress'
+import { resolve } from 'path'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: ' ',
```

Comment:
Why is there no title?
```suggestion
  title: ' REMSFAL Dokumentation',
```